.eslintrc

```diff
@@ -48,6 +48,10 @@
     "no-throw-literal": 2,
     "no-trailing-spaces": 2,
     "no-unused-expressions": 0,
+    "no-unused-vars": [2, {
+      "argsIgnorePattern": "^var_|opt_|unused_|_",
```

Comment:
filed a feature request, ill see if i can submit the PR myself

https://github.com/greggman/eslint-plugin-google-camelcase/issues/3